.circleci/config.yml

```diff
@@ -270,7 +303,8 @@ workflows:
           requires:
             - wagon
             - rhel_wagon
-            - integration_tests
+            - integration_tests_505
+            - integration_tests_510
```

Comment:
Let's not use `*all_jobs` for nightlies anymore, since we don't want to run release. All others we should run.